tests/common/config/unit/test_config.py

```diff
@@ -0,0 +1,144 @@
+import unittest
+
+import os
+
+import ims.common.config as config
+import ims.exception.config_exceptions as config_exceptions
+from ims.common.log import trace
+
+
+class TestBadPath(unittest.TestCase):
+    """ Tests whether an exception is raised when loading from invalid path """
+
+    @trace
+    def setUp(self):
+        self.old_path = os.getenv('BMI_CONFIG')
+        os.environ['BMI_CONFIG'] = "blahblah.cfg"
+
+    def runTest(self):
+        with self.assertRaises(IOError):
+            config.load(force=True)
+
+    def tearDown(self):
```

Comment:
Hi, @chemistry-sourabh , one suggestion here, can we have a base class, which contains interface for setUp, tearDown, so that other specific test classes can implement base class and override method in base class. Also, we don't need repetitively write tearDown function in every test case. What do you think of it?